official-antistasi-community/A3-Antistasi
# Vanilla DLC Weapons

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds conditional checks for adding marksmen DLC rifles and MGs to thew vanilla templates.
Minor adjustments and tweaks to the regular loadouts, such as colour matching using dlc attachments which are not restricted.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

Each tampered template has been loaded with all DLC enabled - no issues reported in RPT or by the template validator.

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Enable DLCs, load templates, observe enemy machine gunners, snipers, and marksmen.

********************************************************
Notes:
